.vscode/tasks.json

```diff
@@ -119,6 +119,30 @@
         },
         "problemMatcher": "$msCompile",
         "group": "build"
+    },
+    {
+      "label": "build language server",
+      "command": "dotnet",
+      "type": "shell",
+      "args": [
+        "build",
+        "-c",
+        "Debug",
+        "src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj"
+      ],
+      "problemMatcher": "$msCompile",
+      "group": "build"
+    },
+    {
+      "label": "launch vscode with language server",
+      "command": "${execPath}",
+      "type": "process",
+      "options": {
+        "env": {
+          "DOTNET_ROSLYN_LANGUAGE_SERVER": "${workspaceRoot}/artifacts/bin/Microsoft.CodeAnalysis.LanguageServer/Debug/net8.0/Microsoft.CodeAnalysis.LanguageServer.dll"
```

Comment:
I think the env var might have been deleted since this PR was created, not seeing it in vscode-csharp - https://github.com/search?q=repo%3Adotnet%2Fvscode-csharp%20DOTNET_ROSLYN_LANGUAGE_SERVER&type=code

Not opposed to adding it back for this use case though

I really wish there was a way to create a profile and launch vscode with that profile from the CLI, then we could just use vscode settings instead of an env var.

Reply:
ahh its a different name - https://github.com/search?q=repo%3Adotnet%2Fvscode-csharp%20DOTNET_ROSLYN_SERVER_PATH&type=code